### PR TITLE
英文翻译优化

### PR DESCRIPTION
**Pull request by C29H25N3O5**

### CODE REVIEW

File: parts/language/lang_en.lua [hunk 1]
```diff
@@ -3,21 +3,21 @@ return{
...
-    playedTooMuch="You have been playing for too long! Playing a block game is fun, but it's time to take a break.",
+    playedLong="You have been playing for a long time. Take a break!",
+    playedTooMuch="You have been playing for far too long! Techmino is fun, but remember to take some breaks!",
 
     atkModeName={"Random","Badges","K.O.s","Attackers"},
-    royale_remain="$1 Players Remaining",
+    royale_remain="$1 Players Remains",
     powerUp={[0]="+000%","+025%","+050%","+075%","+100%"},
-    cmb={nil,"1 Combo","2 Combo","3 Combo","4 Combo","5 Combo","6 Combo","7 Combo","8 Combo","9 Combo","10 Combo!","11 Combo!","12 Combo!","13 Combo!","14 Combo!!","15 Combo!!","16 Combo!!","17 Combo!!!","18 Combo!!!","19 Combo!!!","MEGACMB"},
+    cmb={nil,"1 Combo","2 Combos","3 Combos","4 Combos","5 Combos","6 Combos","7 Combos","8 Combos","9 Combos","10 Combos!","11 Combos!","12 Combos!","13 Combos!","14 Combos!!","15 Combos!!","16 Combos!!","17 Combos!!!","18 Combos!!!","19 Combos!!!","MEGACOMBO"},
```
Comment (MrZ626) [inline parts/language/lang_en.lua:20]: changing of this line is not neccessary, and it's too long

**Comment by MrZ626** [comment]
For some important reason I push -f to the repository, this PR should be remade.
You can backup the lang_en.lua, update the local repo to recent state, then put the translate file back and PR again

**Comment by MrZ626** [comment]
Or I can copy your translating file then commit on my computer, adding you as co-author if you allow me to do that

**Comment by C29H25N3O5** [comment]
Alright I guess I’ll just need to revise it

**Pull request closed without merging**